_pages/en_US/get-started.txt

```diff
@@ -23,42 +23,110 @@ If you see an unusual menu, STOP - you already have custom firmware! Continue fr
 
 #### Section III - Select a Method
 
-Use the version table below to select a method. A few things to note:
-  + The version table below is *inclusive*. For example, "from 11.4.0 to 11.15.0" includes 11.4.0, 11.15.0, and all versions in between.
-  + Software versions do not work the same as decimals. Versions 11.10.0 and above are newer than 11.3.0, and are therefore not compatible with Soundhax.
-  + The number and letter after the system version are not important.
-  + No matter the method you follow, the end result is the same (boot9strap+Luma3DS custom firmware setup on the latest firmware).
-
-<table>
-  <colgroup>
-    <col span="1" style="width: 20%;">
-    <col span="1" style="width: 40%;">
-  </colgroup>
-  <thead>
-    <tr>
-      <th style="text-align: center">System Version</th>
-      <th style="text-align: center">What to do</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td style="text-align: center; font-weight: bold;">11.16.0 (latest version)</td>
-      <td style="text-align: center; font-weight: bold;"><a href="seedminer">Seedminer</a></td>
-    </tr>
-	  <tr>
-      <td style="text-align: center; font-weight: bold;">11.14.0 to 11.15.0</td>
-      <td style="text-align: center; font-weight: bold;">Update your 3DS to the latest version through System Settings</td>
-    </tr>
-    <tr>
-      <td style="text-align: center; font-weight: bold;">11.4.0 to 11.13.0</td>
-      <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(ssloth-browser)">Installing boot9strap (SSLoth-Browser)</a></td>
-    </tr>
-    <tr>
-      <td style="text-align: center; font-weight: bold;">1.0.0 to 11.3.0</td>
-      <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(soundhax)">Installing boot9strap (Soundhax)</a></td>
-    </tr>
-  </tbody>
-</table>
+To find the correct method for your device, please enter the system version you found in Section II.
```

Comment:
the please isn't strictly necessary but it's whatever

Reply:
rude